ui/tests/test_neonpass.py

```diff
@@ -72,15 +65,17 @@ def context(
 
 
 @pytest.fixture
-def request_sol() -> None:
+def request_sol(solana_url: str) -> None:
```

Comment:
this already handled good in integration tests

Reply:
The same as in another thread.